addons/mail/static/tests/crosstab/crosstab.test.js

```diff
@@ -41,10 +41,12 @@ test("Delete starred message updates counter", async () => {
         res_id: channelId,
         starred_partner_ids: [serverState.partnerId],
     });
+    onRpc("/discuss/channel/messages", () => step("/discuss/channel/messages"));
     const env1 = await start({ asTab: true });
     const env2 = await start({ asTab: true });
     await openDiscuss(channelId, { target: env1 });
     await openDiscuss(channelId, { target: env2 });
+    await assertSteps(["/discuss/channel/messages", "/discuss/channel/messages"]);
```

Comment:
Can't we wait for message to be shown with selector instead?

Reply:
Actually this test fails for a good reason. Deleting a starred message will not update the counter in other tabs if they are not aware of the message. I'm working on a real fix instead of a test fix.